doc/update_source_files_to_2.0.0.sed

```diff
@@ -62,3 +62,11 @@ s/tracer particle/particle/g
 s/tracer/particle/g
 s/Tracer/Particle/g
 s/TRACER/PARTICLE/g
+
+# Rename assembler base class
+s/internal::Assembly::AssemblerLists<dim>/Assemblers::AssemblerLists<dim>/g
+s/internal::Assembly::Assemblers/Assemblers/g
+s/struct AssemblerLists/class Manager/g
+s/AssemblerLists/Manager/g
+s/AssemblerBase/Interface/g
+s:assembly.h:simulator/assemblers/interface.h:g
```

Comment:
we could keep assembly.h (that includes the other files) to avoid this change?!

Reply:
I would vote for removing the file. The interface is broken anyway for anybody who wrote an individual assembler (not many people), and those would be the only ones including assembly.h. interface.h *is* the new assembly.h.